packages/rlc-common/src/buildSamples.ts

```diff
@@ -0,0 +1,835 @@
+import { getClientName } from "./helpers/nameConstructors.js";
```

Comment:
Please add the license information in the new added file.

Reply:
Can you also format this file?  Looks there are some style problems in this file.

Reply:
Let me add the lint & format checking in our ci for this folder. I believe my IDE would format for me.